packages/twenty-front/src/modules/context-store/hooks/useFindManyRecordsSelectedInContextStore.ts

```diff
@@ -36,12 +33,13 @@ export const useFindManyRecordsSelectedInContextStore = ({
   const queryFilter = computeContextStoreFilters(
     contextStoreTargetedRecordsRule,
     contextStoreFilters,
-    objectMetadataItem,
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    objectMetadataItem!,
```

Comment:
We should never resort to non-null assertion, it's like using any.

Reply:
yes sorry, I was testing stuff and forgot about it!
I have introduced another tooling since